jobqueue_features/decorators.py

```diff
@@ -217,6 +217,9 @@ def _submit(self, cluster, client, f, *args, **kwargs):
                 cluster, "ntasks_per_node", None, **kwargs
             )
 
+            executable = kwargs.get("executable", None)
+            if executable:
+                f.__name__ = "fmpi/" + executable
```

Comment:
Is it really "fmpi/"?

Reply:
I was just trying to think of an easy way to differentiate between the forked MPI job and a python task with MPI support. Open to other ideas...

Reply:
OK, now I understand!  :)